phac-nml/irida-next
# chore: Updated view components from 2.82.0 --> 3.5.0

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Updated view components from 2.82.0 --> 3.5.0

The biggest issue was updating all view components children were created with the `with_` prefix.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

No UI changes, this is all behind the scenes. 

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

Check to make sure all pages load.  This is also done through our tests.

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

  the [PR acceptance checklist](https://phac-nml.github.io/irida-next/docs/development/development_processes/code_review#acceptance-checklist) for this PR.
